core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/api/constructors.kt

```diff
@@ -0,0 +1,16 @@
+package org.jetbrains.kotlinx.dataframe.api
+
+import io.kotest.matchers.shouldBe
+import org.junit.Test
+
+class ConstructorsTests {
```

Comment:
Could you maybe also add a test with duplicate column names? Since DynamicDataFrameBuilder seems to be able to handle those